src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -315,14 +319,16 @@ private IContainer calculateGlobalPKAnalysisFor(Simulation simulation, Compound
          var vssPlasmaOverF = createParameter(VssPlasmaOverF, bloodPlasmaPK[Vss], VolumePerBodyWeight);
          var vdPlasma = createParameter(VdPlasma, bloodPlasmaPK[Vd], VolumePerBodyWeight);
          var vdPlasmaOverF = createParameter(VdPlasmaOverF, bloodPlasmaPK[Vd], VolumePerBodyWeight);
-         var vssPhysChem = createParameter(VssPhysChem, calculateVSSPhysChemFor(simulation, compoundName), VolumePerBodyWeight);
+         if(simulation is IndividualSimulation)
```

Comment:
We're only going to calculate the VssPhysChem for individuals for now.

Reply:
The calculation is different from every other pk parameter. It's adding up organ volumes for the individual and dividing by body weight. They body weight parameter is scaled for each individual for all the pk parameter calculations, but organ volumes are not. Basically, the VssPhysChem calculation for each individual is not as simple as the others.

Reply:
OK, I see.
Actually it's because the function `calculateVSSPhysChemFor `lacks the individual id to pick the correct volumes (and other parameters like HCT) from the proper individual.
If it's too much work for this extension right now - can you please create an issue for this?

Reply:
It's more about getting the correct values from the simulation given the IndividualId. Check in VSSCalculator.cs

```
public IParameter VSSPhysChemFor(Simulation simulation, string compoundName)
```
If there's an easy way to pull the required parameters for the individual, I can arrange for it.

Reply:
There is not..the function needs to be rewritten completely. (What we do with. Bodyweight scaling, we have to do for all parameters). We probably need a function tthet does it, reset the parameters etc. 
Just too much work I think

Reply:
I think there is an easy way.. but let's ask @msevestre about that :)

Reply:
Is there an easy way? Am I missing something?

Reply:
@msevestre was of the opinion yesterday that there's not an easy way. I'll open an issue for it anyway. Let's not make it part of this PR.

#2414

Reply:
>Is there an easy way? Am I missing something?

Well, as far as I can see Individual Ids used/generated by the pop runner are always 0,..., N-1 - so an Individual Id can be just used as index when pulling a values array for a parameter?
Which is of course error prone and could be improved, but it's how the population runner works at the moment, right?
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/1227a5c738c63a68b4c6e46bb2d5395d2bdc96b6/src/PKSim.Core/Services/IPopulationExportTask.cs#L150-L163

Reply:
sure that's the theory. But the runner does not do this at all. It creates a table that is then apply via variation in SimModel. The only place were we do something like this is for BW scaling.  

So in short. There is no easy way. Since this was not implemented before, I am perfectly happy to say that all global pk are available except VSS Phys Chem that will be implemented at a later time (or never)